templates/custom/vesta/vesta.html.twig

```diff
@@ -0,0 +1,139 @@
+{#
+/**
+ * @file vesta.html.twig
+ * Default theme implementation to present Vesta data.
+ *
+ * This template is used when viewing Vesta pages.
+ *
+ *
+ * Available variables:
+ * - content: A list of content items. Use 'content' to print all content, or
+ * - attributes: HTML attributes for the container element.
+ *
+ * @see template_preprocess_vesta()
+ *
+ * @ingroup themeable
+ */
+#}
+
+{%
+  set classes = [
+    'detail-layout',
+    'vesta'
+  ]
+%}
+
+<article{{ attributes.addClass(classes) }}>
+  <h1>{{ '@name - openinghours and addresses'|t({'@name': entity.name}) }}</h1>
+
+  <div class="contact-details box-top contact-details-inverted">
+    <div class="inner-box">
+      <h2>{{ 'Contact details'|t }}</h2>
+
+      <ul class="icon-list">
+        {# address #}
+        {% if entity.postalAddress|render %}
+          {% spaceless %}
+            <li>
+              <i class="icon-marker" aria-hidden="true"></i>
+              <span>
+                <strong>{{ entity.name }}</strong>
+                &mdash;
+                {{ entity.postalAddress.street }} {{ entity.postalAddress.houseNumber }}{{ entity.postalAddress.box ? '/' ~ entity.postalAddress.box  }},
+                {{ entity.postalAddress.postalCode }} {{ entity.postalAddress.city }}
+              </span>
+            </li>
+          {% endspaceless %}
+        {% endif %}
+
+        {# landlines #}
+        {% set landlines = entity.contactNumbers.landlines.getIterator() %}
+        {% if landlines|length > 0 %}
+          {% for landline in landlines %}
+            <li>
+              <i class="icon-normalphone" aria-hidden="true"></i>
+              <span>
```

Comment:
What's the use of these spans?

Reply:
In an icon-list, the span is used as a block-element to make the text doesn't wrap under the icon.

so 
```
[icon] I'm a line of text that is wrapping
under the icon
```

VS.

```
[icon] I'm a line of text that is NOT 
       wrapping under the icon
```

Reply:
OK, should have spotted that one in the icon-list i suppose.
It looks funny, you have an inline-element wrapped in an inline-element... but you want it displayed as a block element.

So why not make the a-tag display block or wrap it in a div?

Reply:
Sounds good, but that'll require some refactoring on lots of other components as well and therefore belongs in a seperate PR, don't you think?

[DTGB-644](https://digipolisgent.atlassian.net/browse/DTGB-644) 🎉

Reply:
Agreed. Also refactoring this might take some time... so make a separate ticket and maybe check with the PM if it's okay to do this right now...